### PR TITLE
GDB-4661 Add tooltips and banners on action components and dialogs

### PR DESCRIPTION
**Pull request by svilenvelikov**
* Add tooltips on action elements
* Add save in progress indicator
* Removed obsolete .gitkeep file

https://ontotext.atlassian.net/browse/GDB-4661

### CODE REVIEW

File: src/assets/i18n/en.json [hunk 1]
```diff
@@ -43,7 +60,9 @@
   "MESSAGES": {
     "CONFIRM_MAPPING_DELETION": "Do You want to remove this mapping?",
     "DISCARD_CHANGES": "Changes you made will not be saved.",
-    "CONFIRM_NEW_MAPPING": "All mappings will be lost. Do you wat to proceed?"
+    "CONFIRM_NEW_MAPPING": "All mappings will be lost. Do you wat to proceed?",
```
Comment (desislava-hristova-ontotext) [inline src/assets/i18n/en.json:63]: Maybe you were hungry? :)

Comment (desislava-hristova-ontotext) [reply src/assets/i18n/en.json:63]: I see, Teo was :)

Comment (svilenvelikov) [reply src/assets/i18n/en.json:63]: maybe just blind :)

Comment (svilenvelikov) [reply src/assets/i18n/en.json:63]: I'll fix it in a minute

**Pull request merged**